### PR TITLE
ROX-12391: Revisit public docs about Central/Sensor compatibility

### PR DESCRIPTION
**Pull request by jschnath**
Previous PR closed on accident so here is attempt #2

### CODE REVIEW

File: modules/install-acs-operator.adoc [hunk 1]
```diff
@@ -30,6 +30,12 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
 If you choose automatic updates, when a new version of the Operator is available, Operator Lifecycle Manager (OLM) automatically upgrades the running instance of your Operator.
 +
 If you choose manual updates, when a newer version of the Operator is available, OLM creates an update request. As a cluster administrator, you must then manually approve that update request to update the Operator to the new version.
++
+[IMPORTANT]
+====
+If you choose manual updates, you should update the {product-title-short} Operator in all
+secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed should have the same version to ensure optimal
```
Comment (kcarmichael08) [inline modules/install-acs-operator.adoc:37]:
```suggestion
secured clusters when you update the {product-title-short} Operator in the cluster where Central is installed. The secured clusters and the cluster where Central is installed should have the same version to ensure optimal functionality.
```

**Review by kcarmichael08 — COMMENTED** [review]
Thank you for making the changes - I think one word got left off, which I have commented on. Sorry about the PR closing, our process is a bit convoluted - thank you for your patience!

File: modules/install-acs-operator.adoc (hunk 1 above)
Comment (jschnath) [reply modules/install-acs-operator.adoc]: Whoops, must've cut that off on accident. It's fixed now, thanks 👍

**Comment by kcarmichael08** [comment]
Preview builds:

- http://file.rdu.redhat.com/kcarmich/prs-11.21-jschnath-rox-12391/installing/install-ocp-operator.html#install-acs-operator_install-ocp-operator
- http://file.rdu.redhat.com/kcarmich/prs-11.21-jschnath-rox-12391/upgrading/upgrade-helm.html
- http://file.rdu.redhat.com/kcarmich/prs-11.21-jschnath-rox-12391/upgrading/upgrade-roxctl.html

**Review by kcarmichael08 — COMMENTED** [review]
LGTM, I have requested a merge.

**Comment by kcarmichael08** [comment]
Please merge to `rhacs-docs` and then cherry pick to:

- `rhacs-docs-3.70`
- `rhacs-docs-3.71`
- `rhacs-docs-3.72`
- `rhacs-docs-3.73.0`

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**